File: server/src/main/java/backend/repository/user/UserRepository.java
package backend.repository.user;

import backend.model.user.UserEntity;
import org.springframework.data.jpa.repository.JpaRepository;

public interface UserRepository extends JpaRepository<UserEntity, Long> {
    UserEntity findByNickname(String nickname);
    UserEntity findByUserId(Long userId);
    UserEntity findBySocialId(String socialId);
    // Jwt 인증 테스트용
    boolean existsByNickname(String nickname);
}

File: server/src/main/kotlin/backend/user/UserRepository.kt
package backend.user

import org.springframework.stereotype.Repository
import common.entities.User

@Repository
class UserRepository {

//    fun findUser(id : Long) : User = User(
//        id,
//        "Harry",
//        ""
//    )

    fun findUser(id : Long) : String = "Harry"

}